### DISCUSSION TITLE
Labwc exposes all wayland protocols to all clients

### DISCUSSION THREAD

**Discussion by ghost** [labwc/labwc#2392] [Q&A] [answered]
Hi, I'm working to help clear up a claim that compositors that use wlroots (e.g. labwc) should be avoided due to ability of apps to record each other without permission. One can see an example of this claim here:
https://discuss.privacyguides.net/t/people-should-avoid-wayland-environments-which-use-wlroots-as-the-compositor/17899

...and my response here:
https://discuss.privacyguides.net/t/people-should-avoid-wayland-environments-which-use-wlroots-as-the-compositor/17899/17

Thanks!

**Comment by Consolatis** [comment, marked as answer]
Hardcoded implementation:
- #2398

Tracked for the security context protocol here:
- #1897

Related:
- #1002
- #1004

**Reply by ghost** [reply]
Hi, yes I was on the wlroots IRC chat (Greg_M). Thanks much!
